internal/server/routes.go

```diff
@@ -173,8 +176,8 @@ type routeIdentifier struct {
 // constructed from the get, post, put, del helper functions.
 type routeGroup struct {
 	*gin.RouterGroup
-	noAuthentication bool
-	noOrgRequired    bool
+	authenticationOptional bool
```

Comment:
Nice rename! Negative booleans names are always confusing

Reply:
Nit: `optionalAuthentication` and `optionalOrganization`?

Reply:
I think `optionalOrganization`  would make sense for a field that identifies the organization by ID or name. This is a boolean about if the organization is optional or not. I think putting optional second reads a bit better because it's short for "organization is optional" -> `organizationOptional`